api/v1alpha1/bundle_types.go

```diff
@@ -51,13 +52,41 @@ type BundleSource struct {
 	Type string `json:"type"`
 	// Image is the bundle image that backs the content of this bundle.
 	Image *ImageSource `json:"image,omitempty"`
+	// Git is the git repository that backs the content of this Bundle.
+	Git *GitSource `json:"git,omitempty"`
 }
 
 type ImageSource struct {
 	// Ref contains the reference to a container image containing Bundle contents.
 	Ref string `json:"ref"`
 }
 
+type GitSource struct {
+	// Repository is a URL link to the git repository containing the bundle.
```

Comment:
nit: it may not be clear enough.  My suggestion is
`Repository is a HTTPS URL to clone the git repository containing the bundle.`

Reply:
I can understand the perspective that this description may need to be revisited, but we could in theory add support for any git-based protocol (ssh/https/etc.) that's supported on their side.

Reply:
I see.  I see only the HTTPS URLs in the unit tests so I guessed that that's the only supported format.

Reply:
Currently yes -- we only support publicly available git repositories in this iteration. But we would like to support additional formats as well once we have some form of secret-sharing/authentiation in place.